Gemfile.lock

```diff
@@ -9,8 +9,19 @@ GEM
       zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
+    bibtex-ruby (5.1.6)
```

Comment:
@justanhduc can you remove this file - Or is it needed ?

Reply:
@edgarriba I think it is an automatically generated file so it can be safely deleted. I pushed a new commit but seems like you already merged. Should I make a new pr?